workflow-bot-app/src/main/java/com/symphony/bdk/workflow/api/v1/WorkflowsApi.java

```diff
@@ -126,4 +128,14 @@ ResponseEntity<List<VariableView>> listWorkflowGlobalVariables(
           + "The date is an ISO 8601 date", example = "2022-09-21T15:43:24.917Z")
       @RequestParam(required = false, name = "updated_after") Instant updatedAfter);
 
+
+  @ApiOperation("Upload a secret")
```

Comment:
should these be in `WorkflowsMgtApiController` instead with `@Authorized(headerTokenKey = X_MANAGEMENT_TOKEN_KEY)` ?